### PR TITLE
Add a method to get descriptions for an application's glean pings

### PR DESCRIPTION
**Pull request by wlach**
Needed for https://github.com/mozilla/lookml-generator/pull/143

### CODE REVIEW

File: mozilla_schema_generator/glean_ping.py [hunk 1]
```diff
@@ -134,15 +134,22 @@ def get_probes(self) -> List[GleanProbe]:
...
             dependency_pings = self._get_json(self.ping_url_template.format(dependency))
-            pings |= dependency_pings.keys()
+            ping_data.update(dependency_pings)
+        return ping_data
+
+    def get_pings(self) -> Set[str]:
+        return self._get_ping_data().keys()
 
-        return pings
+    def get_ping_descriptions(self) -> Dict[str, str]:
```
Comment (fbertsch) [inline mozilla_schema_generator/glean_ping.py:148]: Can we add a test for this in `tests/test_glean.py`?

Comment (wlach) [reply mozilla_schema_generator/glean_ping.py:148]: Yep, I was just thinking of that after submitting this. Testing the full dependency resolution seems like a bit more than I really want to take on right now, but I added a quick test for the basic functionality (which actually caught a bug!)

Comment (wlach) [reply mozilla_schema_generator/glean_ping.py:148]: Just to clarify the above comment a bit, lest I appear irresponsible: my preferred medium-term solution here is to provide better endpoints from probeinfo.tmo to reduce the complexity inside *this* repository.

**Review by fbertsch — APPROVED** [review]
I agree about the dependency resolution - I know we've discussed that before. For now though this should work.

**Pull request merged**